utils/find-overlay-dependencies.sh

```diff
@@ -68,16 +68,22 @@ for sdk in ${(k)SDKS}; do
   DEPENDS_ON=()
   printf "%s:\n\t" "$sdk"
   deps=$(echo "@import $1;" | xcrun -sdk $sdk clang -arch $arch -x objective-c - -M -fmodules 2>/dev/null)
+  if [[ $? != 0 ]]; then
+    # Don't `continue` here because we want to clear out the SDK line if it exists
+    echo "unsupported"
+  fi
+
   for overlay in $ALL_OVERLAYS; do
     (echo "$deps" |
         egrep -v 'module.(module)?map' |
+        egrep -v 'os/object.h|os/availability.h|os/base.h|os/lock.h' |
         egrep -v "\b${CUSTOM_NAMED_MODULES[$1]-$1}\b" |
         egrep -q "\b$overlay\b") &&
         DEPENDS_ON+=${CUSTOM_NAMED_MODULES[$overlay]-$overlay}
   done
-  echo " $DEPENDS_ON"
+  echo "$DEPENDS_ON"
   if [[ $UPDATE_CMAKE == 1 ]]; then
-    perl -pi -e "s/^  $CMAKE_DEPENDS_NAME[$sdk].*$/  $CMAKE_DEPENDS_NAME[$sdk] $DEPENDS_ON/" $CMAKE_PATH
+    sed -i "" -e "s/^  $CMAKE_DEPENDS_NAME[$sdk].*$/  $CMAKE_DEPENDS_NAME[$sdk] $DEPENDS_ON/" $CMAKE_PATH
```

Comment:
Let's make this not depend on specific leading whitespace either. How about this?

```
sed -i "" -E -e "s/^([ \t]+$CMAKE_DEPENDS_NAME[$sdk]).*$/\1 $DEPENDS_ON # auto-updated/" "$CMAKE_PATH"
```

Note the final quotes too; please test with a space in the path!